index.bs

```diff
@@ -135,9 +96,148 @@ Of course, you are invited to [contribute]() any feedback, comments, or question
+- requirements needed to permanently be **online**
+- requirements needed to understand the **Scholarly Communication Notification protocol** and **Policies**
+
+With this regard:
+
+: App
+:: The App is a headless Autonomous Agent that doesn't need a Human Agent to execute write operations on the Data Pod.
+It is a trusted application for the Maintainer <s>that could in principle update any artifact on the Data Pod</s>. This application
+could run as a background service on the computer of the maintainer and be online as long as the computer is connected
+to a network.
+: Dashboard
+:: The Dashboard is an Agent that responds to feedback from an Human Agent. Typically this Dashboard runs in a browser
+can be in an online or offline modus when a browser is running on the computer of the maintainer with one of the browser
+tabs opened with the dashboard application.
+: Orchestrator
+:: The Orchestrator is an Autonomous Agent that can read the (Scholarly) **Inbox** of the **Data Pod** and append to
+the **Artefact Lifecycle Event Log**. Guided by policies expressed as business rules, the **Orchestrator** also communicates with the Scholar Community network using
+the **Scholarly Communication Notification protocol** implementing **Policies**.
+
+<img src="images/mellon-orchestrator-solar.svg" width="40%">
+
+The three Actors can be seen as a mini solar system with the **Data Pod** as the Sun. The **App** is a fully trusted agent that runs very near to
+the data pod (in level of trust). The **Orchestrator** has very limited access rights to the data pod. The main
+task of the Orchestrator is to update the event log and execute policies in order to talk the Scholarly Communication Notification
+protocol (read/send messages from/to the corrrect Service Hubs).
+The **Dashboard** sits in between, it has control over the **Data Pod** but might (always) need user input to update the pod.
+Communication between the App/DashBoard and the Orchestrator happens via the Data Pod's Inbox.
+
+The networks below demonstrates the CRUD privileges imagined for the different actors in this document. The first
+network demonstrates a typical Solid setup where the Dashboard is a single page application that has direct access
+to the Data Pod and the App is more a background process running in the same computer as the Maintainer.
+
+<div><img src="images/mellon-crud-app.svg" width="80%"></div>
+<i>CRUD operations in case the DashBoard is single-page application and the App a background task</i>
+
+Issue: Are the Rules and Inbox necesearily part of the Data Pod or can they exist as separate resources?
+
+The second network demonstrates a more classic setup with a browser Dashboard controlled by a server App which uses a
+Data Pod as backend storage.
+
+<div><img src="images/mellon-crud.svg" width="80%"></div>
+<i>CRUD operations in case the Dashoard and App is a classic client/server application</i>
+
+Communication between Data Pod and Scholarly Community network
```

Comment:
I suggest that I and @JWerbrouck add a similar summary section about our use cases.